SlideKitDemo-iOS/SlideKitDemo-iOS.xcodeproj/project.pbxproj

```diff
@@ -375,11 +381,27 @@
 		};
 /* End XCConfigurationList section */
 
+/* Begin XCRemoteSwiftPackageReference section */
+		68642A122D609659008288D1 /* XCRemoteSwiftPackageReference "SlideKit" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/mtj0928/SlideKit";
```

Comment:
Please use SlideKit in the local directory instead of one in GitHub.
The demo apps are one of the kinds of playgrounds of SlideKit.

I would like to provide an experience that if the code changes in the local package, developers can check the changes in the demo apps.